src/html/loading/index.html

```diff
@@ -7,7 +7,7 @@
   <body>
     <div class="container">
       <div class="loader"></div>
-      <h1>Initializing...</h1>
+      <h1>初始化中...</h1>
```

Comment:
@liviavinci This wont solve the problem as English user will see the page again and again. HTML localization usually takes places during rendering. As this page is load directly from backend (which doesnt control localization now), I can come up with two potential solutions:

1. make this string into a React component and localized from React
2. remove this string completely

Reply:
Regarding #1, this is not possible (AFAIK) since this html is not part of the react code at all. It is independent entirely.

Reply:
Why don't we just put in some JS like
```
<h1 id='loading-text'>初始化中...</h1>
<script>
  const locale = window.navigator.userLanguage || window.navigator.language;
  const loadingText = {
    'en-US': 'Initializing...',
    'zh-Hans-CN': '初始化中...',
  }
  document.getElementById('main-loader').innerHTML = loadingText[locale]
</script>
```

Reply:
@alex-cory I am fine with removing the text as loading animation means `intializing...`